go.mod

```diff
@@ -4,9 +4,10 @@ go 1.21.3
 
 require github.com/realdream-ai/gdspx v0.0.0
 
-replace github.com/realdream-ai/gdspx => github.com/realdream-ai/gdspx v0.0.0-20241028072423-3577d7e1307f
+replace github.com/realdream-ai/gdspx => github.com/realdream-ai/gdspx v0.0.0-20241030013253-086e5f846004
 
 require (
+	github.com/realdream-ai/gdspx v0.0.0-20241028072423-3577d7e1307f
```

Comment:
This line seems to be duplicated with `require github.com/realdream-ai/gdspx v0.0.0` above.

Nit: And I think if we merge `require github.com/realdream-ai/gdspx` with `replace github.com/realdream-ai/gdspx ...`, we may omit `replace github.com/realdream-ai/gdspx ...` in `cmd/spx/go.mod` & `cmd/spx/template/go.mod.txt`